docs/capsword.md

```diff
@@ -24,4 +24,4 @@ keyboard.keymap = [
 
 |Key                    |Aliases             |Description                                    |
 |-----------------------|--------------------|-----------------------------------------------|
-|`KC.CW`                |`KC.CAPSWORD`       |Enables/disables capsword                      |
\ No newline at end of file
+|`KC.CW`                |`KC.CAPSWORD`       |Enables/disables CapsWord                      |
```

Comment:
Some question for space after table separator. Is there a style recommendation?

Reply:
The linter I linked above _does not_ have a rule for this. Adding the space looks better IMO, but for this PR I kept the changes that didn't effect what ends up in the final rendered HTML to a minimum.

Reply:
My preference for markdown tables is to align things in the markdown, but then I read that far more often than going to the rendered content :)

It's also far more hassle unless the linter can automatically fix it.